revenue_app/tests/transactions_example.sql

```diff
@@ -0,0 +1,11 @@
+SELECT
```

Comment:
What's this file for? I couldn't find anywhere you're using it in tests.

Reply:
It's used in line 1073